### PR TITLE
Recommendations for minor documentation updates

### PR DESCRIPTION
**Pull request by chivandikwa**
Recommendations for minor documentation updates

- Remove duplicated instructions and pluralize section with multiple operations
- Fix incorrect instruction
- Move a Note section to when it is relevant

- [x] Self review completed
- [x] Changes tested by running Docfx

### CODE REVIEW

File: articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md [hunk 1]
```diff
@@ -72,11 +72,7 @@ On the `Shape` Class add the following Operation:
 
 - `Draw` with no return type.
 
-On the `Canvas` Class add the following Operation:
```
Comment (chivandikwa) [inline articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md:75]: This section is duplicated

image

File: articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md [hunk 2]
```diff
@@ -138,7 +134,7 @@ Finally, create an Association from the `Group` to the `Shape` Class and ensure
   - `Is Collection` is checked.
 - In the `Source End` section that:
   - `Is Collection` is *un*checked.
-  - `Is Nullable` is *un*checked.
```
Comment (chivandikwa) [inline articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md:141]: For the Group to Shape association to be an aggregate association then nullable should be checked on source end for the relationship to become 0..1. This is a typo as the GIF correctly shows this as the desired setting.

File: articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md [hunk 3]
```diff
@@ -184,16 +180,6 @@ Now re-add the classes which were removed from the other Diagram, namely the fol
 
 Lay them out so that `Shape` is located center-top of the diagram, `Triangle` bottom-left and `Circle` bottom-right.
 
-> [!NOTE]
```
Comment (chivandikwa) [inline articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md:187]: This note mentions 4 associations, there will only be two at this stage only later in the steps will this apply for circle and triangle. The context of the note is correct when moved further down

**Comment by dandrejvv** [comment]
Thank you @chivandikwa for submitting this pull-request. I think those fixes make sense and I've merged them in.

**Pull request merged**